packages/core/src/utils/summarizer.code-context.md

```
@@ -0,0 +1,174 @@
+  preserveSignatures?: boolean;
+  preserveImports?: boolean;
+  preserveTypes?: boolean;
+  groupByFile?: boolean;
+  includeComplexity?: boolean;
+}
+```
+
+## Integration with Existing Code
+
+The implementation:
+- Extends existing `llmSummarizer` patterns
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)